lib/pharos/phases/configure_master.rb

```diff
@@ -53,6 +55,8 @@ def install
           copy_external_etcd_certs
         end
 
+        push_cloud_config if @config.cloud&.config
```

Comment:
The master also runs `configure_kubelet` => `ConfigureKubelet`, won't that also upload the same file into place? It doesn't hurt to upload the same file twice, but it seems a little odd.

Maybe this should be at a different path, so that the two phases don't interfere with eachother... or just skip the file write here and rely on the file put into place by the `ConfigureKubelet` phase, but those kinds of inter-phase dependencies are annoying to figure out later on.

Reply:
Changed to use `/etc/pharos/kubelet` for kubelet's cloud-config.